src/SecurityManager.ts

```diff
@@ -284,15 +284,15 @@ export const crossSigningCallbacks: ICryptoCallbacks = {
 };
 
 export async function promptForBackupPassphrase(): Promise<Uint8Array> {
-    let key: Uint8Array;
+    let key!: Uint8Array;
```

Comment:
I wasted some time looking up the reference for this syntax. I can't find anything here: https://www.typescriptlang.org/docs/handbook/variable-declarations.html so I guess this covers it: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator although as far as I understand it this is not an expression. Any idea of a better reference?

Anyway, my question is: is there a reason why you put the `!` at the time of declaration instead of writing `return key!;` later?

Reply:
And why did you prefer this style? My immediate feeling was that doing it at the end felt more natural, so I'm curious to learn.

Reply:
In functions where you always initialise a variable I prefer this approach rather than having to add a `!` to each reference, which in this case is only 1 but often is more.

Reply:
Also, thank you for the link. Side rant: how is this only documented in some release notes???

Reply:
> In functions where you always initialise a variable I prefer this approach rather than having to add a `!` to each reference, which in this case is only 1 but often is more.

Makes sense, thanks.